test/system/components/mentoring/mentor_conversations_list_test.rb

```diff
@@ -19,6 +19,15 @@ class MentorConversationsListTest < ApplicationSystemTestCase
           within("td:nth-child(10)") { assert_text "https://exercism.io/conversations/1" }
         end
       end
+
+      test "paginates results" do
+        visit test_components_mentoring_mentor_conversations_list_url
+        click_on "2"
+
+        within("tbody > tr:first-child") do
+          within("td:nth-child(4)") { assert_text "Tournament" }
```

Comment:
I have added a similar comment to another PR, but maybe consider not verifying the exact index of the td that has the text? Instead maybe do something like "there is a td that has that text"?

Reply:
I've added a commit to the other PR that hopes to fix this issue.